massa-pos-exports/src/controller_traits.rs

```diff
@@ -21,7 +21,10 @@ pub struct Selection {
     pub producer: Address,
 }
 
-#[cfg_attr(any(test, feature = "testing"), mockall::automock)]
+#[cfg(feature = "test-exports")]
+use std::sync::Arc;
```

Comment:
unused ?

Reply:
They are used by `mockall_wrap::wrap` because my macro doesn't import them. The macro is a bit buggy but works as a first version I want to improve it when I will have all the tests re-written